src/RuleSet/Sets/PHP80MigrationSet.php

```diff
@@ -23,6 +23,7 @@ public function getRules()
     {
         return [
             '@PHP73Migration' => true,
+            'no_unneeded_final_method' => true,
```

Comment:
can you put a comment why it's matching PHP 8.0 migration?

Reply:
sure, it is here https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/5219 , I'll add it to the code